R-package/README.md

```diff
@@ -221,6 +251,73 @@ At build time, `configure.win` will be run and used to create a file `Makevars.w
 1. Edit `configure.win` directly
 2. Edit `src/Makevars.win.in`
 
+### Build Precompiled Binaries of the CRAN Package
+
+This section is mainly for maintainers. As long as the R package is not available on CRAN (which will build precompiled binaries automatically) you may want to build precompiled versions of the R package manually, since these will be easier for users to install.
+
+For more details, see ["Writing R Extensions"](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Building-binary-packages).
+
+Packages built like this will only work for the minor version of R used to build them. They may or may not work across different versions of operating systems
```

Comment:
```suggestion
Packages built like this will only work for the minor version of R used to build them. They may or may not work across different versions of operating systems.
```